build/release/Jenkinsfile

```diff
@@ -17,14 +17,15 @@ pipeline {
       steps {
         script {
           docker.build('maven').inside {
+            def publishingUri = "https://central.sonatype.com/publishing/deployments"
             def targetBranch = "${env.BRANCH_NAME}-${env.BUILD_NUMBER}"
             sh "git config --global user.name 'ivy-team'"
             sh "git config --global user.email 'info@ivyteam.ch'"
             sh "git checkout -b ${targetBranch}"
+            sh "git tag -l | xargs git tag -d"
```

Comment:
easier to re-run; if the jenkins checkout already has created the release tag ... by a previously failed run, it may survived and let sub-sequent builds fail. Even if the tag was removed from remote :face_with_open_eyes_and_hand_over_mouth: